src/expression.ts

```diff
@@ -129,7 +129,10 @@ export class Expression<DataType, IsNotNull extends boolean, Name extends string
   }
 
   in<Q extends Query<any>>(
-    array: DataType[] | Expression<DataType, IsNotNull, any> | SpecificQuery<DataType, Q>,
+    array:
+      | ReadonlyArray<DataType>
```

Comment:
It looks like this will only accept a readonly array.  I think it should accept either.  There are some use cases where we get the results of a query, and then use those as an `in` for another query, and we wouldn't be able to make those into readonly arrays.

Reply:
`ReadonlyArray` is a subtype of `Array`. Anything that accepts a `ReadonlyArray` can also accept an `Array`.

(In TS, the `Readonly` on types is a guarantee that the _receiver_ will not modify it. It doesn't constrain what the caller can pass in.)

Reply:
Isn't a default array mutable, and can't be read-only?

Reply:
That's not what `Readonly` means. It's a guarantee that the _receiver_ can't modify it.

[TypeScript playground link](https://www.typescriptlang.org/play/?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtVIAoAPALngCUQpg8IBPAQRhijoB5VkBbAIxBgB8ASjIA3HFmABuAFAyweAM4Z4ReAF54AbQCMAXVlEAdAAdkigBYEATENmJidmcbOWCAZjtA)